diff --git a/S15lib/instruments/timestamp7.py b/S15lib/instruments/timestamp7.py
index cb2cfc1..60486e5 100644
--- a/S15lib/instruments/timestamp7.py
+++ b/S15lib/instruments/timestamp7.py
@@ -19,7 +19,8 @@ from typing import List, Optional, Tuple, Union
 import numpy as np
 import psutil
 
-from ..g2lib import parse_timestamps as parser
+#from ..g2lib import parse_timestamps as parser
+import fpfind.lib.parse_timestamps as parser
 
 
 class TimestampTDC2:
@@ -246,7 +247,7 @@ class TimestampTDC2:
         """
         duration = duration if duration else self.int_time
         self._call_with_duration(["-a1"], duration=duration)
-        t, p = parser.read_a1(self.outfile_path, legacy=self._legacy)
+        t, p = parser.read_a1(self.outfile_path, legacy=self._legacy, ignore_rollover=True)
 
         t1 = t[p & 0b0001 != 0]
         t2 = t[p & 0b0010 != 0]
@@ -350,7 +351,7 @@ class TimestampTDC2:
         """See parser.read_a1 doc."""
         duration = duration if duration else self.int_time
         self._call_with_duration(["-a1"], duration=duration)
-        t, p = parser.read_a1(self.outfile_path, legacy=self._legacy)
+        t, p = parser.read_a1(self.outfile_path, legacy=self._legacy, ignore_rollover=True)
         return t, p
 
     def begin_readevents(
